posts/2021-05-17-metrics-jee9-21006-beta.adoc

```diff
@@ -0,0 +1,178 @@
+
+To begin recieving metrics you will need to configure the request timing thresholds for slow or hung requests. The following example demonstrates enabling request timing metrics for `mpMetrics-3.0`, with a sample threshold configuration for both slow and hung servlet requests.
+
+[source, xml]
+----
+    <featureManager>
+	    <feature>mpMetrics-3.0</feature>
+	    <feature>requestTiming-1.0</feature>
+      <!-- other features omitted for brevity -->
+    </featureManager>
+
+    <requestTiming sampleRate="1" slowRequestThreshold="10s">
+		<servletTiming
+			slowRequestThreshold="2s"
+    		        hungRequestThreshold="10s"/>
+    </requestTiming>
+----
+
+For more information regarding Request Timing with MicroProfile Metrics:
+* link:{url-prefix}/docs/latest/reference/feature/requestTiming-1.0.html[Request Timing Documentation]
+* link:{url-prefix}/docs/latest/introduction-monitoring-metrics.html[Monitoring with Metrics]
+* link:{url-prefix}/docs/latest/jmx-metrics-list.html#request-timing-stats[Request Timing MXBean Documentation]
+
+=== Try it now 
+
+To try out these features, just update your build tools to pull the Open Liberty All Beta Features package instead of the main release. The beta works with Java SE 15, Java SE 11, or Java SE 8.
+
+If you're using link:{url-prefix}/guides/maven-intro.html[Maven], here are the coordinates:
+
+[source,xml]
+----
+<dependency>
+  <groupId>io.openliberty.beta</groupId>
+  <artifactId>openliberty-runtime</artifactId>
+  <version>21.0.0.6-beta</version>
+  <type>pom</type>
+</dependency>
+----
+
+Or for link:{url-prefix}/guides/gradle-intro.html[Gradle]:
+
+[source,gradle]
+----
+dependencies {
+    libertyRuntime group: 'io.openliberty.beta', name: 'openliberty-runtime', version: '[21.0.0.6-beta,)'
+}
+----
+
+Or take a look at our link:{url-prefix}/downloads/#runtime_betas[Downloads page].
+
+[#jakarta]
+== Jakarta EE 9 Beta Features package
+
+As of the 21.0.0.2-beta release, Open Liberty is the first vendor product to be Jakarta EE Web Profile 9.0 compatible. With the Open Liberty 21.0.0.3-beta release, Open Liberty is the first vendor product to be added to the link:https://jakarta.ee/compatibility/#tab-9[Jakarta EE Platform 9.0 compatibility list].
+
+This Open Liberty beta introduces the following Jakarta EE 9 features which now possess their all-new Jakarta EE 9 package names:
+
+* OpenID Connect Client 1.0 (`openidConnectClient-1.0`)
+* OpenID Connect Provider 1.0 (`openidConnectServer-1.0`)
+* Social Media Login 1.0 (`socialLogin-1.0`)
+* Automatic Certificate Management Environment (ACME) Support 2.0 (`acmeCA-2.0`)
+
+The `openidConnectClient-1.0`, `openidConnectServer-1.0`, `socialLogin-1.0` and `acmeCA-2.0` features will automatically adapt to the level of Java EE or Jakarta EE that is already in use, so no change is needed when using them with Jakarta EE 9.
```

Comment:
Link the features to their specific feature page in docs.  For example:
https://openliberty.io/docs/latest/reference/feature/openidConnectClient-1.0.html